### ISSUE TITLE
Chrono Cross hangs after first battle

### ISSUE THREAD

**Issue by Helsionium** [socram8888/tonyhax#127]
**tonyhax version**: tonyhax 1.4.3
**Installation method**: wrote the savegame to the memory card using a modded PS2
**Entry point game**: Tony Hawk's Pro Skater 4 German version (SLES-03955)
**Console model**: SCPH-5502 (PAL)
**Integrity check**: passed
**BIOS version**: v3.0 01/06/97 E
**Target game**: Chrono Cross US version (SLUS-01041)

**Bug explanation**:
Chrono Cross boots and initially seems to run fine.
But for me, the game always hangs after the first battle and won't return back to the dungeon after the battle results screen. The disc drive sounds like it is stuck in an endless loop of spinning down and spinning up again (sound recording attached)
[chrono_cross_drive_sound.zip](https://github.com/socram8888/tonyhax/files/8792843/chrono_cross_drive_sound.zip)

- Tried different media and burn speeds - no change
- Other games (namely Xenogears and Metal Gear Solid) work just fine on the same PS1 using tonyhax
- Chrono Cross (the same media that don't work properly on the PS1) works just fine on a PC using RetroArch with BeetlePSX
- It doesn't matter whether I try to run an unmodified image of Chrono Cross (Disc 1 MD5: 3B29EE931DF29F51ECF0817F30E103D7) or one with a PAL patch applied

**Comment by alex-free** [comment]
This sounds (literally) like a dying CD  drive. Some games are more finicky then others. Just because one game works ok with an iffy CD drive doesn't prove the CD drive is working optimally. This should be looked at though to see if it is reproducible.

**Comment by socram8888** [comment]
Could you please try with latest 1.4.5?

**Comment by Helsionium** [comment]
> Could you please try with latest 1.4.5?

For me, Chrono Cross is working even worse than before. I tried 10 times, but the game always hangs before or after the first in-game FMV (the opening FMV, title screen and gameplay demo worked flawlessly every time, though). When it hangs before the first in-gameFMV (right after naming the first character), it hangs in the manner I described earlier. When it hangs after the FMV, the disc continues to spin normally.

Although Xenogears and Metal Gear Solid still work normally even for longer periods of time, I guess it could be a dying disc drive after all. FMV playback in THPS4 is getting noticeably worse, and that's an original disc which is in good condition. A few times today, the disc stopped spinning entirely and the drive started making funny noises (possibly failed attempts at starting the drive motor).
